### DISCUSSION TITLE
Yet another Symfony RoadRunner runtime

### DISCUSSION THREAD

**Discussion by FluffyDiscord** [roadrunner-server/roadrunner#1867] [Show and tell]
Would you consider listing in docs another Symfony runtime? https://github.com/FluffyDiscord/roadrunner-symfony-bundle

**Comment by rustatian** [comment]
Sure, why not. But you know, w/o PR (roadrunner-server/docs) it would be hard 😄

**Reply by FluffyDiscord** [reply]
Somehow i wasn't aware of that repo, will do
